.github/workflows/docker-moby-latest.yml

```diff
@@ -73,3 +73,63 @@ jobs:
           payload-file-path: "./payload-slack-content.json"
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_DOCKER_LATEST_WEBHOOK }}
+
+  test_latest_compose:
+    strategy:
+      matrix:
+        latest_compose_version: [true, false]
+        latest_compose-spec_version: [true, false]
+        exclude:
+          - latest_compose_version: false
+            latest_compose-spec_version: false
```

Comment:
we don't want to run the tests with the current dependencies, as this is tested on every PR modifying the compose module